revscoring/errors.py

```diff
@@ -75,6 +75,12 @@ def __init__(self, datasources, info=None, arg=None):
         super().__init__("Query failed ({0}:{1})"
                          .format(datasources, info))
 
+class UnexpectedContentType(DependencyError):
+    def __init__(self, text, content_type, arg=None):
+        super().__init__("Expected entity of content type {} "
```

Comment:
I approved this but I just noticed that "Entity" in this message.  Entity is jargon for a specific type of wiki content.  Entity JSON is the format that Items, Properties, and Lexemes are built on top of for Wikidata.  Maybe this could be "Expected content of type {}" where the value could be "Entity JSON".

Reply:
Good point, amended, let me know if it is better now :)